src/BlogPost.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react";
+
+// props: {
```

Comment:
Let's add some prop validation to ensure that our component is receiving the kind of `post` object it expects. This will provide an additional layer of testing within the app. Check out https://reactjs.org/docs/typechecking-with-proptypes.html for an explanation of how this works.